lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3286,8 +3286,6 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
   else
     CmdArgs.push_back(Args.MakeArgString(getToolChain().getThreadModel()));
 
-  Args.AddLastArg(CmdArgs, options::OPT_fveclib);
-
```

Comment:
Why is this moved 1000 lines down?

Reply:
To reuse the `EnableVec` and `VectorizeAliasOption` variables in line 4356.

Reply:
Why not just put the -fveclib change here. Why do we care about the vectorizer command line being present? It's default on all the time, right?

+#ifdef FLANG_LLVM_EXTENSIONS
+  if(Args.getLastArg(options::OPT_fveclib))
+    Args.AddLastArg(CmdArgs, options::OPT_fveclib);
+  else 
+    CmdArgs.push_back("-fveclib=PGMATH");
+#else
+  Args.AddLastArg(CmdArgs, options::OPT_fveclib);
+#endif

Reply:
It doesn't make sense to add the `-fveclib=PGMATH` flag when there is no vectorization taking place.